cl/opinion_page/templates/view_opinion.html

```diff
@@ -208,7 +208,7 @@ <h2 class="inline">{{ cluster.caption|safe|v_wrapper }}</h2>
             <span class="meta-data-header">Supreme Court Database ID:</span>
             <span class="meta-data-value">
                 {% if cluster.scdb_id %}
-                    <a href="http://supremecourtdatabase.org/analysisCaseDetail.php?cid={{ cluster.scdb_id }}" target="_blank">
+                    <a href="http://scdb.wustl.eduy/analysisCaseDetail.php?sid=&#38cid={{ cluster.scdb_id }}-01" target="_blank">
```

Comment:
Shoot, you've got two errors here! First, you have `.eduy` for some reason. Second, I think the bit about `sid` can just be left out, like so: `http://scdb.wustl.edu/analysisCaseDetail.php?cid={{ cluster.scdb_id }}-01`. 

For example, here's Roe: http://scdb.wustl.edu/analysisCaseDetail.php?cid=1972-048-01